stl/inc/vector

```diff
@@ -1238,17 +1238,17 @@ public:
             "vector insert iterator outside range");
 #endif // _ITERATOR_DEBUG_LEVEL == 2
 
-        _Adl_verify_range(_First, _Last);
-        auto _UFirst         = _Get_unwrapped(_First);
-        auto _ULast          = _Get_unwrapped(_Last);
+        _STD _Adl_verify_range(_First, _Last);
+        auto _UFirst         = _STD _Get_unwrapped(_First);
+        auto _ULast          = _STD _Get_unwrapped(_Last);
         const auto _Whereoff = static_cast<size_type>(_Whereptr - _Oldfirst);
         if constexpr (_Is_cpp17_fwd_iter_v<_Iter>) {
             const auto _Length = static_cast<size_t>(_STD distance(_UFirst, _ULast));
-            const auto _Count  = _Convert_size<size_type>(_Length);
+            const auto _Count  = _STD _Convert_size<size_type>(_Length);
             _Insert_counted_range(_Where, _UFirst, _Count);
 #ifdef __cpp_lib_concepts
         } else if constexpr (forward_iterator<_Iter>) {
-            const auto _Length = _To_unsigned_like(_RANGES distance(_UFirst, _ULast));
+            const auto _Length = _STD _To_unsigned_like(_RANGES distance(_UFirst, _ULast));
             const auto _Count  = _Convert_size<size_type>(_Length);

```

Comment:
Should this line also be changed?

Regex-searching for `(?<!_STD )(_Adl_verify_range|_Get_unwrapped|_Convert_size|_To_unsigned_like|_Allocate_at_least_helper|_Uninitialized_\w+|_Destroy_range|_Move_backward_unchecked|_Copy_n_unchecked4)`, I see 75 unchanged occurrences in `<vector>`. Some I understand (e.g. an `initializer_list` always has `size_t` for its `size()`, so we don't need to worry about ADL), but I don't understand why others aren't being changed.

Since a bunch of lines are potentially affected and I don't totally understand what's happening here, I won't push changes. We can deal with this in a followup.

Reply:
The result of `_To_unsigned_like` is always an integer-like type. It either has no associated namespaces - if it's integral - or we control the associated namespaces - if it's integer-class.

I'm concerned that we seem to be developing more and more complex rules to determine when to `_STD`-qualify calls; maybe we should just qualify everything?

Reply:
BTW I was asking about qualifying `_Convert_size` here, since the occurrence immediately above was changed.

At this point I think we should just qualify everything, if we want to defend against this ADL thing. It's too hard to enforce consistency otherwise. Same rationale as `_STD` qualifying all pretty calls regardless of arguments or lack thereof.

Reply:
> BTW I was asking about qualifying `_Convert_size` here, since the occurrence immediately above was changed.

Yes - I understood this, I was explaining that we don't _need_ to qualify the `_Convert_size` call since we control the argument type and can ensure that ADL is "safe". 

> At this point I think we should just qualify everything, if we want to defend against this ADL thing. It's too hard to enforce consistency otherwise. Same rationale as `_STD` qualifying all pretty calls regardless of arguments or lack thereof.

+1 from me.